storybook/cypress/e2e/autosuggest.cy.ts

```diff
@@ -15,7 +15,7 @@ describe("Autosuggest", () => {
     cy.checkA11y("dso-autosuggest");
     cy.get("@listbox").get("li[role='option']").should("have.length", 10);
 
-    // cy.percySnapshot();
+    cy.get("#root-inner").matchImageSnapshot();
```

Comment:
Dit zal wel moeten omdat de suggesties anders buiten het screenshot vallen.

Hoe verhoudt dit tot meteen `cy.matchImageSnapshot();` ?

Reply:
![Autosuggest -- should show suggestions after input diff](https://github.com/dso-toolkit/dso-toolkit/assets/4646174/f558ba23-48ff-47a8-828d-b37f474a9077)